src/Razor/src/Microsoft.VisualStudio.LiveShare.Razor/Guest/ProjectSnapshotSynchronizationService.cs

```diff
@@ -11,7 +11,7 @@
 
 namespace Microsoft.VisualStudio.LiveShare.Razor.Guest
 {
-    internal class ProjectSnapshotSynchronizationService : ICollaborationService, IAsyncDisposable
+    internal class ProjectSnapshotSynchronizationService : ICollaborationService, System.IAsyncDisposable
```

Comment:
Fixes [VSTHRD112 Implement `System.IAsyncDisposable`](https://github.com/Microsoft/vs-threading/blob/main/doc/analyzers/VSTHRD112.md).

Note that this type used to implement `Microsoft.VisualStudio.Threading.IAsyncDisposable`. I believe this should be fine to change since this class is `internal`.